build.gradle

```
@@ -5,9 +5,10 @@ buildscript {
     ext.jna_version = '5.2.0'
     ext.android_gradle_plugin_version = '3.6.0'
 
-    // N.B. try to keep these two in sync
-    ext.android_components_version = '24.0.0'
-    ext.glean_version = '21.3.0'
+    // Since the Glean version depends on the Android components version,
+    // it is very important to use a modern version of Glean and, ideally,
+    // let this come the embedder product itself.
```

I think you meant ... ?  If I'm wrong, maybe just clarify as you see fit?

```suggestion
    // let this embed the product itself.
```

---

I think it was supposed to be 
```suggestion
    // let this come from the embedding product itself.
```

---

Whoops, my Italianenglish failed me today. @travis79 is correct :D Sorry @mdboom!